Sources/MapboxNavigation/StatusView.swift

```diff
@@ -52,30 +52,57 @@ public class StatusView: UIControl {
     `Status` is a struct which stores information to be displayed by the `StatusView`
      */
     public struct Status {
+        /**
+        `identifier` is a unique string identifier for a `Status`
+         */
         public var identifier: String
-        let title: String
-        var spinner: Bool = false
-        let duration: TimeInterval
-        var animated: Bool = true
-        var interactive: Bool = false
-        var priority: Priority
+        /**
+        `title` is the text that will appear on the `Status`
+         */
+        public let title: String
+        /**
+        `spinner` indicates whether a spinner should be shown during animations
+         set to `false` by default
+         */
+        public var spinner: Bool = false
+        /**
+        `duration` designates the length of time a `Status` will be displayed in seconds
+         To display a `Status` indefinitely, set `duration` to `.infinity`
+         */
+        public let duration: TimeInterval
+        /**
+        `animated` indicates whether showing and hiding of the `Status` should be animated
+         set to `true` by default
+         */
+        public var animated: Bool = true
+        /**
+        `interactive` indicates whether the `Status` should respond to touch events
+         set to `false` by default
+         */
+        public var interactive: Bool = false
+        /**
+        `priority` is a struct which is used to rank a `Status` by importance
+         A lower `priority` value corresponds to a higher priority
+         */
+        public var priority: Priority
     }
     
     /**
      `Priority` is used to display `Status`es by importance
+     Lower values correspond to higher priority.
      */
-    public struct Priority: RawRepresentable {
-        public typealias RawValue = Int
+    public struct Priority {
+        public typealias Priority = Int
```

Comment:
Replace the struct with the typealias. The `value` property will go away: you’ll be able to use an Int as if it were a Priority.

The previous version with the raw-representable Priority would’ve been appropriate if we wanted to define some preset values like `high` and `low` that were tied to specific values. But only a bare typealias is necessary if we’re treating it as just an ordinary number with special semantics.